tests/common.h

```diff
@@ -19,7 +19,17 @@ void display (void);
 
 struct TestMaterial
 {
-  TestMaterial();
+  TestMaterial()
+    : sObj()
+    , CurrentObj(0)
+    , stppObjects()
+  {
+#ifdef NON_STP_BV_OBJECTS
+  nonSTPBV=1;
+#else
+  nonSTPBV=0;
+#endif // NON_STP_BV_OBJECTS
```

Comment:
This could be in the initialization list. If you are going the "variable" road though I'm not sure of the benefit of the macro over a constructor parameter + `const` member.